embed_video/backends.py

```diff
@@ -356,7 +356,7 @@ class VimeoBackend(VideoBackend):
 
     re_detect = re.compile(r"^((http(s)?:)?//)?(www\.)?(player\.)?vimeo\.com/.*", re.I)
     re_code = re.compile(
-        r"""vimeo\.com/(video/)?(channels/(.*/)?)?(?P<code>[0-9]+)""", re.I
+        r"""vimeo\.com/(video/)?(channels/(.*/)?)?((.+)/review/?)?(?P<code>[0-9]+)""", re.I
```

Comment:
Shouldn't `review/?` be just `review/` here?

Is there a situation where the URL is `review123` and not `review/123`?

Overall this regex looks like cluttered and hard to read. It would be best to enumerate the different URLs in a case or comment that demonstrates the different formats the regex should match. 

The different URLs should ideally be part of an unit test case.

Reply:
@aleksihakli Agreed that it's cluttered, but figured this wasn't the time to refactor that since it's only one more case.  Good catch on the accidentally-optional trailing `/`, which I've made required (https://github.com/jazzband/django-embed-video/pull/115/commits/7c23e074a6ab5e4c17e16fbbb3e88928ce4dbf83), and on needing a test case - which has been added (https://github.com/jazzband/django-embed-video/pull/115/commits/08b01766caec699233a98c49b0bebd57c4eeca2f).